jekyll/_cci2/release/configure-release-markers.adoc

```diff
@@ -0,0 +1,50 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Configure release markers
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to configure release markers in your workflow
+:icons: font
+:experimental:
+
+This tutorial shows how to add a release marker step to your workflow config. Release markers allow you log all new releases in one place and link back to the CI/CD pipelines that caused them.
+
+Release markers provide a lightweight way to log your releases without requiring a full CircleCI releases setup. You can use release markers independently, without installing the release agent. If you are interested in comprehensive release visibility and management features, refer to the xref:set-up-circleci-releases#[Set up CircleCI releases] guide for more information.
+
+== 1. Create environment integration
+
+Before adding a release marker step to your workflow, you will need to create a new environment integration in the CircleCI Releases UI.
+
+. In the CircleCI web app, select **Releases** in the sidebar.
+. If this is your first environment setup, select btn:[Create your first Environment Integration]. If you already have environments set up, choose the **Environments** tab and select btn:[Create Environment Integration].
+. Enter a name for your environment, and a description if you would like.
+. Use the dropdown menu to choose your environment integration type, then select btn:[Next: Release Agent Setup]. **You do not need to continue with installing a release agent at this point**, but you will need to reference this environment integration name as part of your config when adding the `log release` step below.
+
+== 2. Update configuration
+
+Next you will update your CircleCI configuration file. Add a `circleci run log release` command to your deployment job. This tells CircleCI to log the release marker in the Releases UI.
+
+Add a `log release` step at the end of your _deployment_ job, for example:
+
+[,yml]
+----
+jobs:
+  deploy-my-service:
+    executor: some-executor
+    steps:
+      ...
+      (existing deployment commands)
+      ...
+      - circleci run release log --environment-name=<some-environment-name> --component-name=<some-component-name> --target-version=<some-version-name>
```

Comment:
Just wondering, for the target version, as this changes for each release would people use a parameter or something to set that?

Reply:
Here's an example of how we handle this internally - https://github.com/circleci/command-convenience-containers/blob/bb7ade65ec37105c5b775590e1c614c0b9d0dc95/deploy/bin/plan-release#L9